libs/net/dal/Migrations/20231014003235_1.0.75.cs

```diff
@@ -0,0 +1,63 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using TNO.DAL;
+
+#nullable disable
+
+namespace TNO.DAL.Migrations
+{
+    /// <inheritdoc />
+    public partial class _1075 : SeedMigration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            PreUp(migrationBuilder);
+            migrationBuilder.AlterColumn<string>(
```

Comment:
This isn't good as it highlights your prior migration was incorrect.  I don't think it will break anything thankfully because the change will not cause a conflict.  

If you make a change after generating a migration you need to rollback first and remove the migration.  Then create a new migration.

`make db-rollback n=1.0.73`
`make db-remove`
`make db-add n=1.0.74`
`make db-update`

Then make sure your rollback works.
`make db-rollback n=1.0.73`
`make db-update`